pkg/internal/loader/disk/subscriptions.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2024 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package disk
+
+import (
+	"context"
+	"sort"
+
+	subv1api "github.com/dapr/dapr/pkg/apis/subscriptions/v1alpha1"
```

Comment:
```suggestion
"github.com/dapr/dapr/pkg/apis/subscriptions/v1alpha1"
```
Just a thought - Since its already in the `subscriptions.go` file I think we can just use `v1alpha1`. `subv1api` could be misinterpreted. or even just `v1api`?

Reply:
Yeah, using the bare package name makes sense to me.